### ISSUE TITLE
Triode's Switch Player plugin function

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#81]
```
This is a feature I use a lot for moving the playback from room to room and is pretty
much the remaining reason why I switch back to the logitech controller app on a daily
basis.
Would love to see this listed in the squeezer home menu.
Thanks.

```

Original issue reported on code.google.com by `drdocmatt` on 2014-02-15 11:54:30

**Comment by nikclayton** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer

**Comment by kaaholst** [comment]
Fixed in 2.0.0
